.github/kubeval.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPTPATH=$(dirname "$0")
+KUBEVAL_VERSION="0.15.0"
```

Comment:
See previous comment about the version

Reply:
This is the version of kubeval we are using. not k8s